<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabWindow</class>
 <widget class="QWidget" name="tabWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_window_settings">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Window settings:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>511</width>
     <height>695</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="horizontalSpacing">
     <number>30</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QCheckBox" name="box_screen_default">
      <property name="text">
       <string>screen</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_screen_default">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="box_fullscreen">
      <property name="text">
       <string>fullscreen</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="box_keep_open">
      <property name="text">
       <string>keep-open</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_keep_open">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>always</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="box_force_window">
      <property name="text">
       <string>force-window</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_force_window">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>immediate</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="box_ontop">
      <property name="text">
       <string>ontop</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="box_no_border">
      <property name="text">
       <string>no-border</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="box_on_all_workspaces">
      <property name="text">
       <string>on-all-workspaces</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="box_autofit">
      <property name="text">
       <string>autofit</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="valuebox_autofit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="box_autofit_larger">
      <property name="text">
       <string>autofit-larger</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="valuebox_autofit_larger">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="box_autofit_smaller">
      <property name="text">
       <string>autofit-smaller</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="valuebox_autofit_smaller">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QCheckBox" name="box_cursor_autohide">
      <property name="text">
       <string>cursor-autohide</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QComboBox" name="comboBox_cursor_autohide">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>number</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>always</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QCheckBox" name="box_cursor_autohide_fs_only">
      <property name="text">
       <string>cursor-autohide-fs-only</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QCheckBox" name="box_no_fixed_vo">
      <property name="text">
       <string>no-fixed-vo</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QCheckBox" name="box_no_keepaspect">
      <property name="text">
       <string>no-keepaspect</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QCheckBox" name="box_no_keepaspect_window">
      <property name="text">
       <string>no-keepaspect-window</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QCheckBox" name="box_monitoraspect">
      <property name="text">
       <string>monitoraspect</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="valuebox_monitoraspect">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QCheckBox" name="box_no_stop_screensaver">
      <property name="text">
       <string>no-stop-screensaver</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QCheckBox" name="box_no_window_dragging">
      <property name="text">
       <string>no-window-dragging</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QCheckBox" name="box_x11_name">
      <property name="text">
       <string>x11-name</string>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QLineEdit" name="valuebox_x11_name">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QCheckBox" name="box_x11_netwm">
      <property name="text">
       <string>x11-netwm</string>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QComboBox" name="comboBox_x11_netwm">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>auto</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
